features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/oracle.sql

```diff
@@ -2913,6 +2913,8 @@ CREATE TABLE AM_KEY_MANAGER (
 )
  /
 
+-- AM_GW_PUBLISHED_API_DETAILS & AM_GW_API_ARTIFACTS are independent tables for Artifact synchronizer feature which --
+-- should not have any referential integrity constraints with other tables in AM database--
 CREATE TABLE IF NOT EXISTS AM_GW_PUBLISHED_API_DETAILS (
```

Comment:
did we add the new table to multi_dc scripts as well?

Reply:
Yes . https://github.com/wso2/carbon-apimgt/pull/8934